documentation/staging/content/quickstart/create-domain.md

```diff
@@ -2,57 +2,49 @@
 title: "Create a domain"
 date: 2019-02-22T15:44:42-05:00
 draft: false
-weight: 6
+weight: 3
 ---
 
+1.  Select a user name and password, following the required rules for password creation (at least 8 alphanumeric characters with at least one number or special character).
```

Comment:
Select a user name and password for the WebLogic domain administrator credentials...  

(Or simply combine this step with the next one.)